projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-data-selector.spec.ts

```diff
@@ -203,7 +299,11 @@ describe("Pivot data selector", () => {
         expectConfigToMatchPanels(null); // pass an invalid type (null) to test for values
     });
 
-    xit("should fire event handlers on reorder in a panel using drag and drop gestures", () => {
+    it("should fire event handlers on reorder in a panel using drag and drop gestures", () => {
+        window.scrollTo(1000, 1000)
```

Comment:
Maybe we can use something like `scrollIntoView` on the target element so that we don't have to scroll an arbitrary number, which may have to change if the size of the window changes?

Reply:
It shouldn't change if the size of the window change since the pivot data selector is around 800px anyway, but I will try if it work with it